docs/en/04_Changelogs/4.0.0.md

```diff
@@ -1051,6 +1052,26 @@ A very small number of methods were chosen for deprecation, and will be removed
 * `ChangeSet` and `ChangeSetItem` have been added for batch publishing of versioned dataobjects.
 * `DataObject.table_name` config can now be used to customise the database table for any record.
 * `DataObjectSchema` class added to assist with mapping between classes and tables.
+* `DBMoney` values are now treated as empty only Amount is null. Values without Currency
```

Comment:
Updated to
```
* `DBMoney` values are now treated as empty only if `Amount` field is null. If an `Amount` value
  is provided without a `Currency` specified, it will be formatted as per the current locale.
```